### PR TITLE
Update error message look

### PR DESCRIPTION
**Pull request by hershilpatel**
Replaces the default html/javascript alert message with error message on the page.  
Screen Shot 2020-12-14 at 8 16 05 AM

Screen Shot 2020-12-14 at 8 17 38 AM

### CODE REVIEW

File: src/main/webapp/WEB-INF/templates/authorization.html [hunk 1]
```diff
@@ -16,16 +16,20 @@
...
-
-        <br>
-
-        <div id="scopes" class="well"></div>
-
-        <br>
-
-        <button id="submit" type="button" class="btn btn-info">Authorize</button>
+    <div class="container">
+        <div style="display: none" class="card card-body bg-danger text-white" id="errorMessage"> </div>
```
Comment (okeefm) [inline src/main/webapp/WEB-INF/templates/authorization.html:20]: On the `revoke-token` page this div is inside the `pageContent` div, whereas here it's not. Was there a reason this was placed outside of `pageContent` on this page?

Comment (okeefm) [reply src/main/webapp/WEB-INF/templates/authorization.html:20]: Or, conversely, that it was in `pageContent` on the other page

File: src/main/webapp/WEB-INF/templates/revoke-token.html [hunk 2]
```diff
@@ -40,10 +46,11 @@
...
       $(document).ready(function() {
 
-
-      	$('#revokeTokenButton').click(function() {
-      	
-      		console.log("Revoke Button Pressed");
+      	$('#revokeTokenButton').click(function() {      	
+			$('#errorMessage').hide();
+			$('#errorMessage').removeClass('bg-danger');
+			$('#errorMessage').removeClass('bg-info');
```
Comment (okeefm) [inline src/main/webapp/WEB-INF/templates/revoke-token.html:52]: I think you could, in theory, chain these methods (and other places that these get called in series), so jQuery doesn't have to do the selector 3 times in a row.
```suggestion
			$('#errorMessage').hide();
			                               .removeClass('bg-danger');
			                               .removeClass('bg-info');
```

File: src/main/webapp/WEB-INF/templates/authorization.html (hunk 1 above)
Comment (hershilpatel) [reply src/main/webapp/WEB-INF/templates/authorization.html:20]: updated to be consistent (error message outside of pageContent)

File: src/main/webapp/WEB-INF/templates/revoke-token.html (hunk 2 above)
Comment (hershilpatel) [reply src/main/webapp/WEB-INF/templates/revoke-token.html]: Chained it in multiple places in new commit!

**Review by okeefm — APPROVED** [review]
Looks good!

**Review by arscan — APPROVED** [review]
Looks good.  I'm having an issue revoking a token, which we can address separately.  It could be because this is on an older code branch and hasn't been kept up to date with master?

**Pull request merged**